### ISSUE TITLE
Wrong labels

### ISSUE THREAD

**Issue by benjavides** [devnag/pytorch-generative-adversarial-networks#12]
On line 152 in `gan_pytorch.py` it says:
>  d_fake_error = criterion(d_fake_decision, Variable(torch.zeros([1,1])))  # zeros = fake

it should say

...torch.zeros([**0**,**0**]))

right?

**Comment by benjavides** [comment]
My error, torch.zeros is creating the vector of 0's, the parameters are setting the dimension
